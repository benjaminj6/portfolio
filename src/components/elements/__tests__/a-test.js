/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'

import { A } from '../'

test('Anchor snapshot', () => {
  const tree = shallow(<A>Test Link</A>)
  expect(tree).toMatchSnapshot()
})
