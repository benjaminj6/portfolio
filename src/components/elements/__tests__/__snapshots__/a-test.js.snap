// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<a class="jsx-3226047878 jsx-3226047878 A">
  Test Link
  <JSXStyle
    styleId="3226047878"
    css=".jsx-3226047878{color:var(--accent-primary);}a.jsx-3226047878:hover{color:var(--white);background:var(--accent-primary);}"
  >
  </JSXStyle>
</a>

`;
